lib/db/schema/patch-018-019.sql

```diff
@@ -0,0 +1,199 @@
+
+CREATE TABLE devices (
+  uid BINARY(16) NOT NULL,
+  id INT UNSIGNED NOT NULL,
```

Comment:
If you delete the device with the current `MAX(id)`, won't that id be used again when you attach a subsequent device?